### PR TITLE
Prepare 1.0 release

### PR DESCRIPTION
**Pull request by OsvaldoRosado**

### CODE REVIEW

File: Library/Sender.ts [hunk 1]
```diff
@@ -101,9 +101,14 @@ class Sender {
                         if (res.statusCode === 200) {
                             setTimeout(() => this._sendFirstFileOnDisk(), this._resendInterval);
                             // store to disk in case of burst throttling
-                        } else if (res.statusCode === 206 ||
```
Comment (KamilSzostak) [inline Library/Sender.ts]: what happened with `206`?

File: package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "applicationinsights",
   "license": "MIT",
   "bugs": "https://github.com/Microsoft/ApplicationInsights-node.js/issues",
-  "version": "0.22.0",
+  "version": "1.0.0",
```
Comment (KamilSzostak) [inline package.json]: YEAH!1!!! 🎆🍾💥🥇👍 🍺

File: Library/Sender.ts (hunk 1 above)
Comment (KamilSzostak) [reply Library/Sender.ts]: Nevermind, I've just noticed the comment.

File: package.json (hunk 2 above)
Comment (OsvaldoRosado) [reply package.json]: 🎉 🎉

**Pull request merged**